src/MOI/indicator_constraint.jl

```diff
@@ -0,0 +1,138 @@
+
+function MOI.supports_constraint(
+    ::Optimizer,
+    ::Type{MOI.VectorAffineFunction{Float64}},
+    ::Type{<:MOI.IndicatorSet{A, S}}
+) where {
+    A, S <: Union{MOI.LessThan{Float64}, MOI.GreaterThan{Float64}, MOI.EqualTo{Float64}}
+}
+    return true
+end
+
+function MOI.is_valid(
+        model::Optimizer,
+        c::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}, S}
```

Comment:
Here and throughout, indentation is off.

Reply:
changed in last commit